### PR TITLE
nethoscope: init at 0.1.1

### PR DESCRIPTION
**Pull request by 0x4A6F**
###### Motivation for this change
Add [nethoscope](https://github.com/vvilhonen/nethoscope).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 115325` at b0c42f05 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/4e17801f723df29ed7565e72f9abf183">nethoscope</a></li></li>
  </ul>
</details>

File: pkgs/tools/networking/nethoscope/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  meta = with lib; {
+    description = "Listen to your network traffic";
+    longDescription = ''
+      Employ your built-in wetware pattern recognition and
+      signal processing facilities to understand your network traffic.
+    '';
+    homepage = "https://github.com/vvilhonen/nethoscope";
+    license = licenses.isc;
+    maintainers = with maintainers; [ _0x4A6F ];
+    platforms = platforms.linux;
```
Comment (siraben) [inline pkgs/tools/networking/nethoscope/default.nix:44]:
```suggestion
    platforms = platforms.unix;
```

**Review by siraben — CHANGES_REQUESTED** [review]

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 115325` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>nethoscope</li>
  </ul>
</details>

File: pkgs/tools/networking/nethoscope/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  meta = with lib; {
+    description = "Listen to your network traffic";
+    longDescription = ''
+      Employ your built-in wetware pattern recognition and
+      signal processing facilities to understand your network traffic.
+    '';
+    homepage = "https://github.com/vvilhonen/nethoscope";
+    license = licenses.isc;
+    maintainers = with maintainers; [ _0x4A6F ];
+    platforms = platforms.unix;
```
Comment (siraben) [inline pkgs/tools/networking/nethoscope/default.nix:44]: Oops, I forgot but I think `buildRustPackage` does not need `meta.platforms.` @SuperSandro2000 please confirm.
```suggestion
```

**Comment by Lassulus** [comment]
Needs a rebase

**Pull request merged**